api/runtime/sandbox/v1/sandbox.proto

```diff
@@ -62,6 +62,7 @@ message CreateSandboxRequest {
 	string bundle_path = 2;
 	repeated containerd.types.Mount rootfs = 3;
 	google.protobuf.Any options = 4;
+	containerd.types.Platform platform = 5;
```

Comment:
Since CRI is in charge for platform, should we remove `Sandbox.Platform`?

Reply:
Remove the platform field from the Sandbox object? How would we thread it through to be able to pass it here https://github.com/containerd/containerd/pull/8059/files#diff-a172c76f5f37d6ebb7881bfc181c224e7075260ff5984d5ddd8375f79fa979adR112-R116? The controller interface only passes an ID and it fills in most of the rest of the data with whats stored on disk/in the sandbox store

Reply:
Sorry, confusing wording.
I meant that we no longer have to provide [Platform](https://github.com/containerd/containerd/blob/cf7b705dcdadf95891b4e25a833a505e1d0e7718/api/runtime/sandbox/v1/sandbox.proto#L42) endpoint on shim side.
Originally we needed it to generate OCI spec for container with respect to target platform [here](https://github.com/containerd/containerd/blob/cf7b705dcdadf95891b4e25a833a505e1d0e7718/pkg/cri/sbserver/container_create.go#L167), but since we already know platform on CRI, shim no longer needs to report that, right?

Reply:
Sorry for delay. No I agree, that was part of the discussion we originally had on your cross platform OCI spec change, if we tell the runtime what platform to care about, then having the runtime return us this feels odd. We store the platform in the sandbox store here, so any containers being launched can easily retrieve it. I didn't want to touch your change here just in case there was some use for it, but I can remove; this change was rather large so didn't want to de-rail with removal of a recent change also

Reply:
Removed the Platform request